etc/sql/oracle/schema.sql

```diff
@@ -1050,10 +1050,12 @@ CREATE INDEX "REQUESTS_RULEID_IDX" ON "REQUESTS" ("RULE_ID") COMPRESS 1 ONLINE t
 
 CREATE INDEX "REQUESTS_EXTERNALID_UQ" ON "REQUESTS" ("EXTERNAL_ID") ONLINE tablespace ATLAS_RUCIO_TRANSIENT_DATA01;
 
+ALTER TABLE REQUESTS ADD requested_at DATE;
 ALTER TABLE REQUESTS ADD submitted_at DATE;
 ALTER TABLE REQUESTS ADD started_at DATE;
 ALTER TABLE REQUESTS ADD transferred_at  DATE;
 
+ALTER TABLE REQUESTS_HISTORY ADD requested_at DATE;
```

Comment:
The table definition should be updated directly imo. I don't know why it was like that actually

Reply:
We have a complete mixture of that at the moment. I will go through it at some point and compile everything to table signatures and remove the alter statements. But in the meanwhile, just add it to the signature as well please.

Reply:
I volunteer to do the clean up in the schema.sql

Reply:
Great! Thanks JJ; I will create a ticket.